packages/angular_devkit/core/src/jobs/README.md

```diff
@@ -0,0 +1,460 @@
+# Description
+
+Jobs is the Angular DevKit subsystem for scheduling and running generic functions with clearly 
+typed inputs and outputs. A Job is code (ie. a function) that can be scheduled, synchronized and
+schedule other jobs. Those Jobs can be run in parallel or follow an execution strategy. They can
+also be extended to allow for transformations, combination and refactoring through different 
+operations.
+
+The whole API is serializable, allowing someone to use a Node Stream or message channel to
+communicate between the job and the job scheduler.
+
+Jobs are lazy, cold and guaranteed to be only executed once per scheduling. Subscribing to a job
+will return events from the point where the job is at.
```

Comment:
cold, and

returns events (*avoid future tense*)
once per scheduling -- not sure what you mean by this. Is a job executed on a recurring schedule (once in each cycle), or executed only once when it is scheduled?

what is "the point where the job is at"? point in what? in the schedule? 
how do you know what point a job is at?